apps/server/src/stores/runtimeState.ts

```diff
@@ -325,12 +319,11 @@ export function updateLoaded(event?: PlayableEvent): string | undefined {
 /**
  * Used in situations when we want to hot-reload all events without interrupting timer
  */
-export function updateAll(rundown: Rundown, timedEventsOrder: EntryId[]) {
-  const timedEvents = filterTimedEvents(rundown, timedEventsOrder);
+export function updateAll(rundown: Rundown, metadata: RundownMetadata) {
   // TODO(remove public): we dont need to make the timedEvents object, we pass primitives and let the functions handle it
```

Comment:
question: can we simplify the function as described in the comment?
otherwise, should we remove the comment?